app.py

```diff
@@ -1,7 +1,22 @@
-from flask_api import FlaskAPI
+from flask_api import FlaskAPI 
+from flask import request
 
 app = FlaskAPI(__name__)
 
+RESULT = {
+        's1' : 'value1',
+        's2' : 'value2',
+        's3' : 'value3',
+        's4' : 'value4',
+        's5' : 'value5'
+}
+#  quesion 1 finshed  
+@app.route('/data', methods=['GET'])
+def query_string():
```

Comment:
`query string` is a special name. using it for function name can be very confusing